test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/main.py

```diff
@@ -1178,6 +1178,20 @@ def _validate_argument_spec(self, docs, spec, kwargs, context=None):
         deprecated_args_from_argspec = set()
         doc_options = docs.get('options', {})
         for arg, data in spec.items():
+            restricted_argument_names = ('message', 'syslog_facility')
+            if arg.lower() in restricted_argument_names:
```

Comment:
You also want to check the option's aliases.

Reply:
Also, you only need to check this for top-level options (i.e. if `not context` is true) IMO. Suboptions can still use these names. Or is that incorrect?